heroku.yml

```diff
@@ -0,0 +1,5 @@
+build:
+  docker:
+    web: Dockerfile.web
```

Comment:
Does heroku allow you to have dockerfiles within a subdirectory? I'd prefer us to have our Dockerfiles within the respective apps, like how explorer has a `Dockerfile` within `explorer/`

Reply:
It does, but doesn't work with the monorepo. It assumes the root dir is the one with the `Dockerfile`. It also needs to map the convention with the `.web` extension for the process name.

This should only be temporary until we get that epic finished fwiw.